src/NetworkEntities.js

```diff
@@ -16,20 +15,34 @@ class NetworkEntities {
 
   createRemoteEntity(entityData) {
     NAF.log.write('Creating remote entity', entityData);
+    const { networkId, template, components } = entityData;
 
-    var networkId = entityData.networkId;
-    var el = NAF.schemas.getCachedTemplate(entityData.template);
+    try {
+      const exists = document.getElementById(networkId);
+      const el = exists || NAF.schemas.getCachedTemplate(template);
+
+      if (!el) {
+        throw new Error(`Failed to create element for template: ${template}`);
+      }
```

Comment:
`NAF.schemas.getCachedTemplate(template)` already log an error if there is no template, so just return here maybe?